lib/bugsnag/configuration.rb

```diff
@@ -36,6 +37,9 @@ class Configuration
     attr_accessor :auto_capture_sessions
     attr_accessor :track_sessions
     attr_accessor :session_endpoint
+    attr_accessor :automatic_breadcrumb_types
+
+    attr_reader :max_breadcrumbs
```

Comment:
If you're starting to adopt yardoc, it would be good to add documentation/comments to new methods that you add/create.

Reply:
To keep things consistent, I was going to add Yardoc comments to new files, and keep existing files as they are, and raise a ticket to Yardoc the whole repo.

Reply:
For the sake of these actually getting written, i've added the yardoc comments now and will continue to in all the other changes.